src/System.Collections.Concurrent/tests/Performance/System.Collections.Concurrent.Performance.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <IncludePerformanceTests>true</IncludePerformanceTests>
+    <ProjectGuid>{16568C86-E97E-42C6-B683-65A1B5AF2EC8}</ProjectGuid>
+    <DisableTests Condition="'$(TargetGroup)' == 'uap' AND ('$(ArchGroup)' == 'arm' OR '$(ArchGroup)' == 'arm64')">true</DisableTests>
```

Comment:
nit: I don't think the `$(TargetGroup) == uap` is necessary since now we are adding configurations.props with netcoreapp as the only configuration.

Reply:
Ok, I'll remove.  I just copied this .csproj from System.Collections (sounds like it should be updated as well).

Reply:
Yes, we should remove in all the Perf .csproj that is present. We can do that over time, it doesn't hurt anything, I just like clean .csproj's :)